### ISSUE TITLE
Please add a light theme

### ISSUE THREAD

**Issue by wp9015362** [connectbot/connectbot#163]
Hello,

currently ConnectBot only has a dark theme.

Would there be any chance you could also add a light theme?

It would be much appreciated.

Regards

**Comment by wp9015362** [comment]
You can see some guidelines for a light theme in the Google Material Design guidelines over there:

https://www.google.com/design/spec/style/color.html#color-themes

Regards

**Comment by wp9015362** [comment]
PS:

When the light them is in use, the FG and BG colors for the terminal should also be inverted by default of course. So, FG should be black and BG should be white.

Regards

**Comment by kruton** [comment]
Switching themes at runtime is problematic.

**Comment by wp9015362** [comment]
> @kruton wrote:
> 
> Switching themes at runtime is problematic.

Why?

There are several other (open source) apps which allow to switch between dark and light themes, like for example [Amaze File Manager](https://github.com/arpitkh96/AmazeFileManager), [TextSecure](https://github.com/WhisperSystems/TextSecure), [spaRSS](https://github.com/Etuldan/spaRSS), [Kore](https://github.com/xbmc/Kore), [OsmAnd](https://github.com/osmandapp/Osmand), [primitive ftpd](https://github.com/wolpi/prim-ftpd) and probably many many more...

None of those have issues switching the theme while the app is running AFAICT.

You might want to have a look at [Amaze File Manager](https://github.com/arpitkh96/AmazeFileManager), as the theme switching is working really nice in it.

Anyway, even if the theme switching would cause problems, which it obviously shouldn't as [Amaze File Manager](https://github.com/arpitkh96/AmazeFileManager) proves, then you could simply add a disclaimer to the option, which tells the user that this option requires a restart of the app and/or prompt the user to restart the app.

Regards

**Comment by alescdb** [comment]
I've just implement a Light/Dark in one of my applications.
I had problems using it before setContentView(). It seems preferable to use it before onCreate().

```@Override
  protected void onCreate( Bundle savedInstanceState )
  {
    if(setup.useLightTheme())
      setTheme(R.style.AppTheme_Light);
    super.onCreate(savedInstanceState);
  }
 ```

**Comment by oldherl** [comment]
Any follow-ups for a light color theme?

On an e-ink device, dark background doesn't look right. I'm mainly talking about the main menu and settings menu, because the console itself can be set to light background anyway.

**Comment by balupton** [comment]
yeah, my phone is the boox palma 2 pro which is color eink, so needs everything to be a light theme (the system theme) for things to be usable

**Comment by kruton** [comment]
The current beta has a light and dark theme.

**Comment by balupton** [comment]
confirmation that the beta has a light theme
